src/scss/ClientCustom/Custom.scss

```diff
@@ -0,0 +1,3 @@
+@import "scss/constants.scss";
```

Comment:
If this is only a single file, why add the ClientCustom folder? Filename should be lowercase. Drop client from this cots irrelevant and doesn’t align to our open source project.

Why not just custom.scss or customOverrides.scss or similar?

Reply:
Perfect-o will do